apps/duplication/archive_translate.py

```diff
@@ -29,6 +28,8 @@
 
 package_service = PackageService()
 
+UPDATE_TRANSLATION_METADATA_MACRO = superdesk.app.config.get('UPDATE_TRANSLATION_METADATA_MACRO')
```

Comment:
I think it's better to use the `app.config` only in the service, otherwise this might be resolved before the settings are loaded properly